apps/molgenis-components/src/client/client.ts

```diff
@@ -266,8 +305,12 @@ const fetchTableData = async (
     ? ',search:"' + properties.searchTerms.trim() + '"'
     : "";
 
-  const schemaId = metaData.id;
-  const columnIds = getColumnIds(schemaId, tableId, metaData, expandLevel);
+  const columnIds = getColumnIds(
+    tableSchemaId,
+    tableId,
+    metaDataMap,
```

Comment:
would we not want instead of metaDataMap to have a 'database' api where we can ask schemas to? Or simply pass the client itself around so we can ask questions to that?

Reply:
metaDataMap is internal to the client, is your suggestion internal or external client api change ( explicitly did not want to change the public client api in this bug fix, suggest separate PR/story for that, as all callers wil need to update)